mag_archiver/mag.py

```diff
@@ -71,7 +69,7 @@ def attr(self):
         return map_[self]
 
 
-def make_mag_query(start_date: Optional[Pendulum] = None, end_date: Optional[Pendulum] = None,
```

Comment:
All of the type hints for dates should be of type `pendulum.DateTime`, except for `release_date` which should be `pendulum.Date`